packages/app-earn-ui/src/components/atoms/Button/Button.module.scss

```diff
@@ -25,14 +26,19 @@
 .secondaryLarge,
 .neutralLarge {
   height: 48px;
-  min-width: 320px;
+  min-width: 100%;
```

Comment:
I'm afraid that with changes like this one you need to verify whether buttons look good everywhere in app